frontend/src/container/WorkspaceContent.jsx

```diff
@@ -804,6 +807,10 @@ export class WorkspaceContent extends React.Component {
                         parentId={content.parentId}
                         label={content.label}
                         fileName={content.fileName}
+                        modified={content.modified}
+                        lang={props.user.lang}
```

Comment:
lang seems to be passed around a lot, maybe it would be better to connect some more components to reducer user to avoid passing it by props that much